modules/azure/client_factory_test.go

```diff
@@ -0,0 +1,106 @@
+// /*
+
+// This file contains unit tests for the client factory implementation(s).
+
+// */
+
+package azure
+
+import (
+	"os"
+	"testing"
+
+	autorest "github.com/Azure/go-autorest/autorest/azure"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Local consts for this file only
+var GovCloudEnvName = "AzureUSGovernmentCloud"
+var PublicCloudEnvName = "AzurePublicCloud"
+var ChinaCloudEnvName = "AzureChinaCloud"
+var GermanyCloudEnvName = "AzureGermanCloud"
```

Comment:
1. These should be `const`, not `var`, as I don't think they ever have a reason to change?
1. The var names start with capitals, which actually makes them public. If you want them private, you need to make them lower case: e.g., `germanyCloudEnvName`. 
1. That said, as I mentioned in a comment above, shouldn't these be exported publicly so users can use them instead of copy/paste?

Reply:
Made these lower-case, as they're only in support of the test itself. End users would configure these via environment variables (likely set outside of code).